docs/install.rst

```diff
@@ -168,6 +162,45 @@ to build from source, unless you are installing a numbered release. (The
 releases packages have the necessary C files packaged with them, and hence do
 not require Cython.)
 
+Prerequisites for Linux
+-----------------------
+
+On Linux, using the package manager for your distribution will usually be 
+the easiest route. In order to build from source, you'll need the python development package
+for your distro. You can install before proceeding with the setup process using
```

Comment:
"You can install before proceeding with the setup process using"  <- Strike this sentence entirely.  I think it's superfluous.